src/App.tsx

```diff
@@ -1,26 +1,28 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import LogIn from "./routes/login";
+import Main from "./routes/main/main";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  const router = createBrowserRouter([
```

Comment:
This routing is fine as it is, but maybe you can take a look at routing using [react-router-dom](https://www.npmjs.com/package/react-router-dom). A way more modern and clean way of implementing the routing. I am going to leave you an example: maybe the div between router and switch is not needed.
`
<Router>

        <div>

          <Switch>

            <Route path="/" exact component={LoginPage} />

            <Route path = "/register" component = {RegisterPage} />

            <Route path = "/addParking/:lat/:long" component = {AddParking} />

            <Route path = "/findParking/:lat/:long" component = {FindParking} />

            <Route path = "/addParkingAddress" component = {GeocodingAddParking} />

            <Route path = "/findParkingCoordinates" component = {GeocodingFindParking} />

            <Route component={LoginPage} />

          </Switch>

        </div>

      </Router>`